### PR TITLE
Add simple ellipse to available geometric objects

### PR DESCRIPTION
**Pull request by chadqueen**
### Overview

The Geometric Object Utility has many 2D and 3D shapes available. One common shape missing from the utility is a 2D ellipse. 

The submitted ellipse implementation is aligned with the circle implementation; creating a 2D shape around the origin in the xy-plane.

### Details

Example ellipse plot output:
image

### CODE REVIEW

File: pyvista/utilities/geometric_objects.py [hunk 1]
```diff
@@ -1299,6 +1299,42 @@ def Circle(radius=0.5, resolution=100):
...
+
+    Examples
+    --------
+    >>> import pyvista
+    >>> semi_major_axis = 8
+    >>> semi_minor_axis = 4
+    >>> ellipse = pyvista.Ellipse(semi_major_axis, semi_minor_axis)
+    >>> ellipse.plot(show_edges=True, line_width=5)
+    """
+
```
Comment (akaszynski) [inline pyvista/utilities/geometric_objects.py:1329]:
```suggestion
```

File: pyvista/utilities/geometric_objects.py [hunk 2]
```diff
@@ -1299,6 +1299,41 @@ def Circle(radius=0.5, resolution=100):
...
+    -------
+    pyvista.PolyData
+        Ellipse mesh.
+
+    Examples
+    --------
+    >>> import pyvista
+    >>> semi_major_axis = 8
+    >>> semi_minor_axis = 4
+    >>> ellipse = pyvista.Ellipse(semi_major_axis, semi_minor_axis)
```
Comment (akaszynski) [inline pyvista/utilities/geometric_objects.py:1326]:
```suggestion
    >>> ellipse = pyvista.Ellipse(semi_major_axis=8, semi_minor_axis=4)
```

File: pyvista/utilities/geometric_objects.py [hunk 3]
```diff
@@ -1299,6 +1299,41 @@ def Circle(radius=0.5, resolution=100):
...
+
+    Parameters
+    ----------
+    semi_major_axis : float, optional
+        Semi-major axis of ellipse.
+
+    semi_minor_axis : float, optional
+        Semi-minor axis of ellipse.
+
+    resolution : int, optional
```
Comment (akaszynski) [inline pyvista/utilities/geometric_objects.py:1313]:
```suggestion
    semi_major_axis : float, default: 0.5
        Semi-major axis of ellipse.

    semi_minor_axis : float, default: 0.2
        Semi-minor axis of ellipse.

    resolution : int, default: 100
```

**Comment by akaszynski** [comment]
Thanks for adding this. Needs a unit test added in `tests/test_geometric_objects.py`.

I made some fairly trivial doc style changes. We're moving towards using `default` instead of `optional` in this library, and in the meantime there is going to be some inconsistency.

**Review by akaszynski — APPROVED** [review]
Nice work. LGTM.

**Review by akaszynski — APPROVED** [review]
LGTM. Thanks!

**Review by tkoyama010 — APPROVED** [review]
LGTM

**Pull request merged**